core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/limit/LimitOpDesc.scala

```diff
@@ -31,7 +31,7 @@ class LimitOpDesc extends OperatorDescriptor {
       "Limit",
       "Limit the number of output rows",
       OperatorGroupConstants.UTILITY_GROUP,
-      inputPorts = List(InputPort("testPortName")),
+      inputPorts = List(InputPort("")),
```

Comment:
@sadeemsaleh what is the difference between the empty string and no parameter? Maybe we should just remove the parameter?